contracts/src/v0.8/vrf/test/FixtureVRFCoordinatorV2_5.t.sol

```diff
@@ -0,0 +1,120 @@
+    hex"a0434d9e47f3c86235477c7b1ae6ae5d3442d49b1943c2b752a68e2a47e247c701";
+  bytes32 internal constant VRF_KEY_HASH = hex"9f2353bde94264dbc3d554a94cceba2d7d2b4fdce4304d3e09a1fea9fbeb1528";
+
+  BlockhashStore internal s_bhs;
+  ExposedVRFCoordinatorV2_5 internal s_coordinator;
+
+  // Use multiple consumers because VRFV2PlusConsumerExample cannot have multiple pending requests.
+  uint256 internal s_subId;
+  VRFV2PlusConsumerExample internal s_consumer;
+  VRFV2PlusConsumerExample internal s_consumer1;
+
+  MockLinkToken internal s_linkToken;
+  MockV3Aggregator internal s_linkNativeFeed;
+
+  function setUp() public virtual override {
+    BaseTest.setUp();
+    vm.stopPrank();
+
+    vm.startPrank(OWNER);
+    s_bhs = new BlockhashStore();
+
+    // Deploy coordinator.
+    s_coordinator = new ExposedVRFCoordinatorV2_5(address(s_bhs));
+    s_linkToken = new MockLinkToken();
+    s_linkNativeFeed = new MockV3Aggregator(18, 500000000000000000); // .5 ETH (good for testing)
+
+    // Configure the coordinator.
+    s_coordinator.setLINKAndLINKNativeFeed(address(s_linkToken), address(s_linkNativeFeed));
+    vm.stopPrank();
+
+    // Deploy consumers.
+    vm.startPrank(SUBSCRIPTION_OWNER);
+    s_consumer = new VRFV2PlusConsumerExample(address(s_coordinator), address(s_linkToken));
+    s_consumer1 = new VRFV2PlusConsumerExample(address(s_coordinator), address(s_linkToken));
+    vm.stopPrank();
+  }
+
+  function _setUpConfig() internal {
+    vm.prank(OWNER);
+    s_coordinator.setConfig(
+      0, // minRequestConfirmations
+      2_500_000, // maxGasLimit
+      1, // stalenessSeconds
+      50_000, // gasAfterPaymentCalculation
+      50000000000000000, // fallbackWeiPerUnitLink
+      500_000, // fulfillmentFlatFeeNativePPM
+      100_000, // fulfillmentFlatFeeLinkDiscountPPM
+      15, // nativePremiumPercentage
+      10 // linkPremiumPercentage
+    );
+  }
+
+  function _setUpProvingKey() internal {
+    uint256[2] memory uncompressedKeyParts = this._getProvingKeyParts(VRF_UNCOMPRESSED_PUBLIC_KEY);
+    vm.prank(OWNER);
+    s_coordinator.registerProvingKey(uncompressedKeyParts, GAS_LANE_MAX_GAS);
+  }
+
+  function _setUpSubscription() internal {
+    vm.startPrank(SUBSCRIPTION_OWNER);
+    s_subId = s_coordinator.createSubscription();
+    s_coordinator.addConsumer(s_subId, address(s_consumer));
+    s_consumer.setSubId(s_subId);
+    s_coordinator.addConsumer(s_subId, address(s_consumer1));
+    s_consumer1.setSubId(s_subId);
+    vm.stopPrank();
+  }
+
+  // note: Call this function via this.getProvingKeyParts to be able to pass memory as calldata and
+  // index over the byte array.
+  function _getProvingKeyParts(bytes calldata uncompressedKey) public pure returns (uint256[2] memory) {
+    uint256 keyPart1 = uint256(bytes32(uncompressedKey[0:32]));
+    uint256 keyPart2 = uint256(bytes32(uncompressedKey[32:64]));
+    return [keyPart1, keyPart2];
+  }
+
+  function _printGenerateProofV2PlusCommand(
```

Comment:
Can you put a comment above this function to explain why are you generating this command for `generate-proof-v2-plus` script and what are you doing with the output of this script?

Reply:
Added!